Dockerfile

```diff
@@ -15,18 +15,24 @@ ENV INITIAL_MEMORY="512m"
 ENV MAXIMUM_MEMORY="512m"
 ARG OVR=""
 ENV JAVA_OPTS="${JAVA_OPTS} -Xms${INITIAL_MEMORY} -Xmx${MAXIMUM_MEMORY}"
-ENV GEOSTORE_OVR_OPT="-Dgeostore-ovr=file://${CATALINA_BASE}/conf/${OVR}"
-ENV JAVA_OPTS="${JAVA_OPTS} ${GEOSTORE_OVR_OPT}"
+# ENV GEOSTORE_OVR_OPT="-Dgeostore-ovr=file://${CATALINA_BASE}/conf/${OVR}"
+ENV GEOSTORE_OVR_OPT=""
+ENV JAVA_OPTS="${JAVA_OPTS} ${GEOSTORE_OVR_OPT} -Ddatadir.location=${DATA_DIR}"
```

Comment:
GEOSTORE_OVR_OPT

Is commented out and then used two lines later

Is there a copy of the "geostore-ovr" file in the datadir?